go-controller/pkg/libovsdb/ops/chassis.go

```diff
@@ -151,10 +151,10 @@ func CreateOrUpdateChassis(sbClient libovsdbclient.Client, chassis *sbdb.Chassis
 			BulkOp:         false,
 		},
 		{
-			Model:          chassis,
-			OnModelUpdates: onModelUpdatesAllNonDefault(),
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model:            chassis,
+			OnModelMutations: []interface{}{&chassis.OtherConfig, &chassis.Encaps},
```

Comment:
It's a trap! (star wars reference, sorry)

```
				encaps := append(chassis.Encaps, encap.UUID)
				chassis.Encaps = sets.New(encaps...).UnsortedList()
```
change to 
```
chassis.Encaps = encap.UUID
```

Reply:
sigh...